src/GHC/TcPlugin/API.hs

```diff
@@ -99,8 +103,8 @@ module GHC.TcPlugin.API
 
     -- | Use these functions to lookup a module,
     -- from the current package or imported packages.
-  , findImportedModule, fsLit, unpackFS, mkModuleName
-  , unitIdFS, stringToUnitId, pkgQual_pkg
+  , findImportedModule, resolveImport, fsLit, unpackFS, mkModuleName
+  , unitIdFS, stringToUnitId, pkgQualToPkgName
   , Module, ModuleName, FindResult(..), UnitId, PkgQual(..)
```

Comment:
```suggestion
  , Module, ModuleName, FindResult(..), UnitId, PkgQual
```

Given that the constructors are different on different GHC versions, I think it should be kept abstract now?